### ISSUE TITLE
Links don't get rejoined when backspacing newlines

### ISSUE THREAD

**Issue by 1j01** [yabwe/medium-editor#1032]
Making a newline and backspacing it should have the same result as doing nothing.

![medium-editor link splitting](https://cloud.githubusercontent.com/assets/3630663/14130343/3d619a52-f600-11e5-86b7-f26e8f4ebdf5.gif)

Probably any equivalent tags should be rejoined.

**Comment by 1j01** [comment]
Spellcheck seems to handle split `<b>` tags okay, but it would still be good to join them.

![kljhlkjhs](https://cloud.githubusercontent.com/assets/3630663/14157780/adebbfa6-f69a-11e5-83c4-660cb6e8e7ea.gif)

**Comment by nmielnik** [comment]
Good point, merging any adjacent elements (even more than just `<a>`) would be a great addition to the core medium-editor code.
